### PR TITLE
Article text components use relay follow buttons

### PR DESCRIPTION
**Pull request by eessex**
We have been applying the Backbone follow button on the client, despite already fetching the necessary follow data for tooltips. 
This PR updates the text component to display Relay artist-follow buttons wherever the 'artist-follow' class is present on a link.

Related PR: https://github.com/artsy/force/pull/5004

### CODE REVIEW

File: src/Components/Publishing/Sections/Text.tsx [hunk 1]
```diff
@@ -72,53 +79,89 @@ export class Text extends Component<Props, State> {
...
+
+  artistIdToArtistFollow = (artistId: string) => {
+    const data = this.context.tooltipsData
+    if (!data || !data.artists) return null
+    return data.artists[artistId]
+  }
+
+  renderArtistFollowButton = (node: Element) => {
+    // Dont include relay components unless necessary
+    // To avoid 'regeneratorRuntime' error
```
Comment (damassi) [inline src/Components/Publishing/Sections/Text.tsx]: Does this still happen?

Comment (eessex) [reply src/Components/Publishing/Sections/Text.tsx]: Yes, in positron where we don't have relay

**Comment by eessex** [comment]
Added a test, this is now ready to go.

**Comment by damassi** [comment]
LGTM 👍

**Comment by artsyit** [comment]
:rocket: PR was released in v25.3.2 :rocket:

**Pull request merged**